tests/foreman/cli/test_capsule.py

```diff
@@ -0,0 +1,157 @@
+# -*- encoding: utf-8 -*-
+# vim: ts=4 sw=4 expandtab ai
+
+"""Test class for capsule CLI"""
+from robottelo.test import CLITestCase
+
+
+class CapsuleTestCase(CLITestCase):
+    def provision_through_capsule(self):
+        """@Test: User can provision through a capsule
+
+        @Feature: Capsules
+
+        @Setup: Some valid, functional compute resource
+        (perhaps one variation of this case for each supported compute
+        resource type).  Also, functioning capsule with proxy is required.
+
+        @Steps:
+            1. Attempt to route provisioning content through capsule that is
+            using a proxy
+            2. Attempt to provision instance
```

Comment:
Try executing `make docs` (or is it `make docs-html`?). This should throw a warning.